notebooks/code_sharing/operational_deposit/operational_deposit_poc.ipynb

```diff
@@ -0,0 +1,4598 @@
+    "\n",
+    "::: {.callout-tip}\n",
+    "\n",
+    "For access to all features available in this notebook, create a free ValidMind account.\n",
+    "\n",
+    "Signing up is FREE — [**Sign up now!**](https://app.prod.validmind.ai)\n",
+    "\n",
+    ":::\n",
+    "\n",
+    "<a id='toc2_3_'></a>\n",
+    "\n",
+    "![Dataset based test architecture](./dataset_image.png)\n",
+    "![Model based test architecture](./model_image.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pre-requisites\n",
+    "\n",
+    "Let's go ahead and install the `validmind` library if its not already installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# %pip install -q validmind"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"VM_OVERRIDE_METADATA\"] = \"true\"\n",
+    "os.environ[\"VALIDMIND_LLM_DESCRIPTIONS_ENABLED\"] = \"true\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='toc4_'></a>\n",
+    "\n",
+    "## Initialize the client library\n",
+    "\n",
+    "ValidMind generates a unique _code snippet_ for each registered model to connect with your developer environment. You initialize the client library with this code snippet, which ensures that your documentation and tests are uploaded to the correct model when you run the notebook.\n",
+    "\n",
+    "Get your code snippet:\n",
+    "\n",
+    "1. In a browser, log into the [Platform UI](https://app.prod.validmind.ai).\n",
+    "\n",
+    "2. In the left sidebar, navigate to **Model Inventory** and click **+ Register new model**.\n",
+    "\n",
+    "3. Enter the model details, making sure to select **Time Series Forecasting** as the template and **Credit Risk - Underwriting - Loan** as the use case, and click **Continue**. ([Need more help?](https://docs.validmind.ai/guide/register-models-in-model-inventory.html))\n",
+    "\n",
+    "4. Go to **Getting Started** and click **Copy snippet to clipboard**.\n",
+    "\n",
+    "Next, replace this placeholder with your own code snippet:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2024-05-23 16:52:02,511 - INFO(validmind.api_client): Connected to ValidMind. Project: Operational deposit model - Initial Validation (clwhu1lol00pz23iio6n4w8k4)\n"
+     ]
+    }
+   ],
+   "source": [
+    "import validmind as vm\n",
+    "\n",
+    "vm.init(\n",
+    "  api_host = \"https://api.prod.validmind.ai/api/v1/tracking\",\n",
```

Comment:
I'm gonna have to rotate your keys since they have been exposed 😞.

Let's sync up offline to run the credentials detector before changes are pushed.

Reply:
Can you remove these credentials from the notebook?

Reply:
removed. thanks. We should add script for all the checks in custom `git push` command.